pom.xml

```diff
@@ -362,7 +363,7 @@
       <dependency>
         <groupId>org.jctools</groupId>
         <artifactId>jctools-core</artifactId>
-        <version>1.2</version>
+        <version>1.2.1</version>
```

Comment:
The original issue https://github.com/JCTools/JCTools/issues/116 was part of 1.2.1 so I think that's a yes.